core/java/iesi-core/src/main/java/io/metadew/iesi/openapi/OpenAPIGenerator.java

```diff
@@ -52,9 +53,9 @@ public TransformResult transformFromJsonContent(String doc) {
     }
 
 
-    private void checkForMessages(SwaggerParseResult result) {
-        if (result.getMessages() != null) {
-            result.getMessages().forEach(log::warn);
+    private void checkForMessages(List<String> messages) {
+        if (!messages.isEmpty()) {
+            throw new SwaggerParserException(messages.get(0));
```

Comment:
why only return the first message?
Shouldn't all errors be included in the exception